README.md

```diff
@@ -580,6 +580,7 @@ See https://github.com/n1trux/awesome-sysadmin#distributed-filesystems
 
 #### File transfer/synchronization
 
+  * [Files Sharing](https://github.com/axeloz/filesharing) - Open Source and self-hosted files sharing application based on unique and temporary links..  `GPL-3.0` `PHP`
```

Comment:
OK, will fix. FYI I have been using the Makefile so I didn't check the syntax at all, just copied and pasted. Thanks